packages/tds-widget/src/chat/bubble/nol/coupon.tsx

```diff
@@ -1,152 +1,125 @@
+<path d="M4.375 16.75H15.625" stroke="#1B1C1F" stroke-width="1.625" stroke-linecap="round"/>
+</svg>
+`
+
+const getSVG = (svg: string) => {
+  return `data:image/svg+xml;utf8,${encodeURIComponent(svg)}`
 }
 
 const DownloadButton = styled(Button)`
-  color: white;
-  font-size: 14px;
-  font-weight: 700;
-  line-height: 24px;
-  float: right;
-  margin-top: 20px;
   padding: 0;
+  border-radius: 0 13px 13px 0;
+  border-left: 1px solid #ecedf7;
+  background:
+    no-repeat center url(${getSVG(DOWNLOAD_ICON)}),
+    #f7f7ff;
+  width: 39px;
+  height: 144px;
   display: inline-block;
-  border: none;
-  background-color: transparent;
 
-  > svg {
-    margin-top: -2px;
-    width: 16px;
-    height: 16px;
-    padding: 3px;
+  :disabled {
+    background-color: red;
   }
 `
 
+const Badge = styled.div<{ valid: boolean }>`
+  margin-bottom: 2px;
+  padding: 0 6px;
+  width: fit-content;
+  border-radius: 6px;
+  font-size: 11px;
+  font-weight: 700;
+  line-height: 22px;
+  color: ${({ valid, theme }) => (valid ? theme.nol.colorPrimaryNol : 'white')};
+  background-color: ${({ valid }) =>
+    valid ? '#F5F6FF' : 'rgba(0, 0, 0, 0.2)'};
+`
+
 export function NolCouponContentBubble({ coupon, onClick }: CouponBubbleProp) {
   const valid = isAfter(new Date(coupon.period.endAt), new Date())
   return (
-    <CouponContainer>
-      <Circle />
-      <Divider />
+    <CouponContainer valid={valid}>
       <Coupon valid={valid}>
+        <Badge valid={valid}>쉿! 🤫 고객님께만 드려요!</Badge>
         <Text
           css={{
-            color: 'white',
-            fontSize: '14px',
-            fontWeight: 400,
-            lineHeight: '24px',
+            color: 'inherit',
+            fontSize: '30px',
+            fontWeight: 700,
+            lineHeight: '38px',
+            marginBottom: '6px',
           }}
         >
-          {format(subDays(new Date(coupon.period.endAt), 1), 'yy.MM.dd')}
-          까지 사용가능
+          {formatNumber(coupon.discount.value)}원
         </Text>
         <Text
           css={{
-            color: 'white',
-            fontSize: '38px',
+            color: 'inherit',
+            fontSize: '12px',
             fontWeight: 700,
-            lineHeight: '44px',
-            height: '44px',
+            lineHeight: '14px',
+            marginBottom: '2px',
+          }}
+        >
+          중복사용가능
+        </Text>
+        <Text
+          css={{
+            color: 'inherit',
+            fontSize: '12px',
+            fontWeight: 400,
+            lineHeight: '14px',
+            marginBottom: '2px',
           }}
         >
-          {formatNumber(coupon.discount.value)}
-          <span css={{ color: 'white', fontSize: '14px', marginLeft: '3px' }}>
-            원
-          </span>
+          {valid
+            ? `${format(subMinutes(new Date(coupon.period.endAt), 1), 'yyyy.MM.dd(HH.mm)')}까지 사용`
```

Comment:
```suggestion
            ? `${format(subMinutes(new Date(coupon.period.endAt), 1), 'yyyy.MM.dd(HH:mm)')}까지 사용`
```

시간 노출 포맷 변경되어야 하겠어요.!